integreat_cms/cms/models/abstract_content_translation.py

```diff
@@ -579,6 +583,63 @@ def save(self, *args: Any, **kwargs: Any) -> None:
             self.last_updated = timezone.now()
         super().save(*args, **kwargs)
 
+    @transaction.atomic
+    def cleanup_autosaves(self) -> None:
+        """
+        Delete all ``AUTO_SAVE``s older than the second last manual save
+        and renumber all affected versions to be continuous.
+
```

Comment:
```suggestion
```
Apparently, the empty line makes the doc build fail.

Reply:
Thanks for the hint! :)